index.html

```diff
@@ -527,6 +527,76 @@ <h3>GpgVerificationKey2020</h3>
   </pre>
   </section>
 
+  <section>
+    <h2>Workday Signature Suite</h2>
+
+    <h3>WorkEd25519Signature2020</h3>
+
+    <table class="simple">
+      <thead>
+        <tr>
+          <th colspan=2>Summary</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>Identifiers</td>
+          <td>WorkEd25519Signature2020</td>
+        </tr>
+        <tr>
+          <td>Status</td>
+          <td>PROVISIONAL</td>
+        </tr>
+        <tr>
+          <td>Authors</td>
+          <td>Workday</td>
+        </tr>
+        <tr>
+          <td>Specification</td>
+          <td><a href="https://github.com/w3c-ccg/ld-cryptosuite-registry/blob/gh-pages/doc/workday/signature-suite.md">Workday Signature Suite 2020</a></td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h3>WorkEd25519VerificationKey2020</h3>
```

Comment:
Possibility to allow other key types in the future here... also depending on how you are representing key types, you can add support for existing suites key types, by being explicit.

here are some suites which have key types that would be compatible:

```
{
      "id": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
      "type": "JwsVerificationKey2020",
      "controller": "did:example:123",
      "publicKeyJwk": {
        "crv": "Ed25519",
        "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ",
        "kty": "OKP",
        "kid": "_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A"
      }
    },
```

```
 {
      "id": "did:key:z6MkuVK4wPm1fFaovcHxRTP6S8chkQuNCdvzuVZHfzVJWGU7#z6MkuVK4wPm1fFaovcHxRTP6S8chkQuNCdvzuVZHfzVJWGU7",
      "type": "Ed25519VerificationKey2018",
      "controller": "did:key:z6MkuVK4wPm1fFaovcHxRTP6S8chkQuNCdvzuVZHfzVJWGU7",
      "publicKeyBase58": "G342M9WaKi6Lp7TFjtRFb34hvqdWnkgeDUeMqiXHb3gj"
    }
```